netstandard/src/ApiCompatBaseline.net461.txt

```diff
@@ -5,6 +5,8 @@ MembersMustExist : Member 'System.Array.Fill<T>(T[], T)' does not exist in the i
 MembersMustExist : Member 'System.Array.Fill<T>(T[], T, System.Int32, System.Int32)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Array.Reverse<T>(T[])' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Array.Reverse<T>(T[], System.Int32, System.Int32)' does not exist in the implementation but it does exist in the contract.
+CannotRemoveAttribute : Attribute 'System.Reflection.DefaultMemberAttribute' exists on 'System.ArraySegment<T>' in the contract but not the implementation.
```

Comment:
Why are you getting all these new entries?

Reply:
They're new checks as of https://github.com/dotnet/arcade/commit/c6e4b4bb3e3aab02e542d16891295fe18c466def, and we haven't been running APICompat on these guys since the checks were added